app/api/items/update_item_string.go

```diff
@@ -0,0 +1,91 @@
+package items
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/go-chi/render"
+
+	"github.com/France-ioi/AlgoreaBackend/app/database"
+	"github.com/France-ioi/AlgoreaBackend/app/formdata"
+	"github.com/France-ioi/AlgoreaBackend/app/service"
+)
+
+// UpdateItemStringRequest is the expected input for item's strings updating
+type UpdateItemStringRequest struct {
+	// Nullable fields are of pointer types
+	LanguageID  int64   `json:"language_id" sql:"-" validate:"language_id"`
+	Title       string  `json:"title" sql:"column:sTitle" validate:"max=200"`        // max length = 200
+	ImageURL    *string `json:"image_url" sql:"column:sImageUrl" validate:"max=100"` // max length = 100
+	Subtitle    *string `json:"subtitle" sql:"column:sSubtitle" validate:"max=200"`  // max length = 200
+	Description *string `json:"description" sql:"column:sDescription"`
+}
+
+func (srv *Service) updateItemString(w http.ResponseWriter, r *http.Request) service.APIError {
+	var err error
+	user := srv.GetUser(r)
+	err = user.Load()
+	if err == database.ErrUserNotFound {
+		return service.InsufficientAccessRightsError
+	}
+	service.MustNotBeError(err)
+
+	itemID, err := service.ResolveURLQueryPathInt64Field(r, "item_id")
+	if err != nil {
+		return service.ErrInvalidRequest(err)
+	}
+
+	input := UpdateItemStringRequest{}
+	formData := formdata.NewFormData(&input)
```

Comment:
Not sure it is recommended to have a variable with such a close name from the package. May be misleading in `formData.ParseJSONRequestData(r)`? `inputData` instead maybe?

Reply:
renamed 'formData' into 'data'